site-docker.yml.sample

```diff
@@ -54,6 +55,7 @@
         - containerized_deployment | bool
         - not (is_atomic | bool)
         - (not (inventory_hostname in groups.get('clients', [])) or (inventory_hostname == groups.get('clients', [''])|first))
+    - role: ceph-validate
```

Comment:
I think the `when` should be removed so we are sure to run this all the time

Reply:
Ok, I've removed it.